docs/pages/modules/config-plugin-and-native-module-tutorial.mdx

```diff
@@ -112,65 +146,70 @@ In another terminal window, compile and run the example app:
   ]}
 />
 
-We should see a screen with a text saying `API key: api-key`. Now let's develop the plugin to inject our custom API key.
+You should see a screen with the text "API key: api-key".
 
-## 4. Creating a new config plugin
+</Step>
 
-Let's start developing our new config plugin. Plugins are synchronous functions that accept an `ExpoConfig` and return a modified `ExpoConfig`. By convention, these functions are prefixed by the word `with`. We will name our plugin `withMyApiKey`. Feel free to call it whatever you like, as long as it follows the convention.
+<Step label="4">
 
-Here is an example of how a basic config plugin function looks:
+## Creating a new config plugin
+
+Start developing your new config plugin. Plugins are synchronous functions that accept an `ExpoConfig` and return a modified `ExpoConfig`. By convention, these functions are prefixed with the word `with`. Name your plugin `withMyApiKey` or use a different name, as long as it follows this convention.
```

Comment:
```suggestion
Plugins are synchronous functions that accept an `ExpoConfig` and return a modified `ExpoConfig`. By convention, these functions are prefixed with the word `with`. Name your plugin `withMyApiKey` or use a different name, as long as it follows this convention.
```

Let's omit the first line since the last line in this section adds the context for the user-facing actionable steps. A concept is being explained here, so let's not mix the two different content types.